.ci/build-freebsd.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -ex
+
+# Pull all the submodules except llvm
+# Note: Tried to use git submodule status, but it takes over 20 seconds
+# shellcheck disable=SC2046
+git submodule -q update --init $(awk '/path/ && !/llvm/ { print $3 }' .gitmodules)
+
+# XXX Drop after Travis upgrades FreeBSD to 12.2 (see also .ci/install-freebsd.sh)
+case $(${CXX:-c++} --version) in
+    *version\ 8.0.*)
+	fetch https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/libcxx-10.0.0.src.tar.xz
+	tar xf libcxx-10.0.0.src.tar.xz
+	export CC=clang10 CXX=clang++10
+	export CXXFLAGS="$CXXFLAGS -nostdinc++ -isystem $PWD/libcxx-10.0.0.src/include"
```

Comment:
Looks like `-nostdinc++` didn't have any affect

Reply:
`-nostdinc++` here makes sure Clang doesn't accidentally fall back to `/usr/include/c++/v1`  which on FreeBSD 12.1 contains libc++ 8.0.1 headers.
<details><summary>Example error</summary>

```c++
FAILED: rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/System.cpp.o
/usr/local/bin/clang++10  -DGLX_GLXEXT_PROTOTYPES -DGL_GLEXT_PROTOTYPES -DHAVE_ALSA -DHAVE_FAUDIO -DHAVE_LIBEVDEV -DHAVE_PULSE -DHAVE_VULKAN -DHAVE_X11 -DLLVM_AVAILABLE -DNDEBUG -DVK_USE_PLATFORM_WAYLAND_KHR -I../rpcs3 -I../ -I../3rdparty/yaml-cpp/include -I../3rdparty/pugixml/src -I../3rdparty/span/include -I../3rdparty/FAudio/src -I../3rdparty/FAudio/include -I/usr/local/include/SDL2 -I/usr/local/include/AL -I../3rdparty/stblib -I../llvm/include -I3rdparty/llvm_build/include -I../asmjit/src -I../3rdparty/cereal/include -I../3rdparty/GL -I../Vulkan/glslang/SPIRV/.. -I../Vulkan/glslang/glslang/.. -I../Vulkan/spirv-tools/include -I../Vulkan/spirv-headers/include -I../3rdparty/xxHash -isystem /usr/local/include/libevdev-1.0 -isystem /usr/local/include -isystem libcxx-10.0.0.src/include -Wl,--exclude-libs,ALL -O3 -DNDEBUG   -pthread -Wall -fno-exceptions -ftemplate-depth=1024 -msse -msse2 -mcx16 -Werror=old-style-cast -Werror=sign-compare -Werror=reorder -Wno-unused-variable -Wno-unknown-pragmas -Wno-invalid-offsetof -Wno-unused-function -Wno-attributes -Wno-comment -fconstexpr-steps=16777216 -Wno-sometimes-uninitialized -Wno-unused-lambda-capture -Wno-unused-private-field -Wno-self-assign -Wno-pessimizing-move -Wno-delete-non-virtual-dtor -Wno-unused-command-line-argument -std=gnu++2a -MD -MT rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/System.cpp.o -MF rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/System.cpp.o.d -o rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/System.cpp.o -c ../rpcs3/Emu/System.cpp
In file included from ../rpcs3/Emu/System.cpp:1:
In file included from ../rpcs3/stdafx.h:23:
In file included from ../Utilities/types.h:6:
In file included from /usr/local/llvm10/lib/clang/10.0.0/include/x86intrin.h:15:
In file included from /usr/local/llvm10/lib/clang/10.0.0/include/immintrin.h:18:
In file included from /usr/local/llvm10/lib/clang/10.0.0/include/xmmintrin.h:27:
In file included from /usr/local/llvm10/lib/clang/10.0.0/include/mm_malloc.h:13:
In file included from libcxx-10.0.0.src/include/stdlib.h:100:
libcxx-10.0.0.src/include/math.h:773:12: error: no member named
      'labs' in the global namespace; did you mean 'abs'?
  return ::labs(__x);
         ~~^
libcxx-10.0.0.src/include/math.h:772:39: note: 'abs' declared here
inline _LIBCPP_INLINE_VISIBILITY long abs(long __x) _NOEXCEPT {
                                      ^
libcxx-10.0.0.src/include/math.h:777:12: error: no member named
      'llabs' in the global namespace
  return ::llabs(__x);
         ~~^
libcxx-10.0.0.src/include/math.h:785:12: error: no member named
      'fabsf' in the global namespace
  return ::fabsf(__lcpp_x);
         ~~^
libcxx-10.0.0.src/include/math.h:789:12: error: no member named
      'fabs' in the global namespace; did you mean 'abs'?
  return ::fabs(__lcpp_x);
         ~~^
libcxx-10.0.0.src/include/math.h:772:39: note: 'abs' declared here
inline _LIBCPP_INLINE_VISIBILITY long abs(long __x) _NOEXCEPT {
                                      ^
libcxx-10.0.0.src/include/math.h:794:12: error: no member named
      'fabsl' in the global namespace
  return ::fabsl(__lcpp_x);
         ~~^
libcxx-10.0.0.src/include/math.h:808:34: error: unknown type name
      'ldiv_t'
inline _LIBCPP_INLINE_VISIBILITY ldiv_t div(long __x, long __y) _NOEXCEPT {
                                 ^
libcxx-10.0.0.src/include/math.h:809:12: error: no member named
      'ldiv' in the global namespace
  return ::ldiv(__x, __y);
         ~~^
libcxx-10.0.0.src/include/math.h:812:34: error: unknown type name
      'lldiv_t'
inline _LIBCPP_INLINE_VISIBILITY lldiv_t div(long long __x,
                                 ^
libcxx-10.0.0.src/include/math.h:814:12: error: no member named
      'lldiv' in the global namespace
  return ::lldiv(__x, __y);
         ~~^
libcxx-10.0.0.src/include/math.h:822:93: error: no member named
      'acosf' in the global namespace; did you mean 'acos'?
  ...float       acos(float __lcpp_x) _NOEXCEPT       {return ::acosf(__lcpp_...
                                                              ~~^
libcxx-10.0.0.src/include/math.h:822:46: note: 'acos' declared here
inline _LIBCPP_INLINE_VISIBILITY float       acos(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:823:93: error: no member named
      'acosl' in the global namespace; did you mean 'acos'?
  ...long double acos(long double __lcpp_x) _NOEXCEPT {return ::acosl(__lcpp_...
                                                              ~~^
libcxx-10.0.0.src/include/math.h:822:46: note: 'acos' declared here
inline _LIBCPP_INLINE_VISIBILITY float       acos(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:829:38: error: call to 'acos' is
      ambiguous
acos(_A1 __lcpp_x) _NOEXCEPT {return ::acos((double)__lcpp_x);}
                                     ^~~~~~
libcxx-10.0.0.src/include/math.h:822:46: note: candidate function
inline _LIBCPP_INLINE_VISIBILITY float       acos(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:823:46: note: candidate function
inline _LIBCPP_INLINE_VISIBILITY long double acos(long double __lcpp_x) ...
                                             ^
libcxx-10.0.0.src/include/math.h:834:93: error: no member named
      'asinf' in the global namespace; did you mean 'asin'?
  ...float       asin(float __lcpp_x) _NOEXCEPT       {return ::asinf(__lcpp_...
                                                              ~~^
libcxx-10.0.0.src/include/math.h:834:46: note: 'asin' declared here
inline _LIBCPP_INLINE_VISIBILITY float       asin(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:835:93: error: no member named
      'asinl' in the global namespace; did you mean 'asin'?
  ...long double asin(long double __lcpp_x) _NOEXCEPT {return ::asinl(__lcpp_...
                                                              ~~^
libcxx-10.0.0.src/include/math.h:834:46: note: 'asin' declared here
inline _LIBCPP_INLINE_VISIBILITY float       asin(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:841:38: error: call to 'asin' is
      ambiguous
asin(_A1 __lcpp_x) _NOEXCEPT {return ::asin((double)__lcpp_x);}
                                     ^~~~~~
libcxx-10.0.0.src/include/math.h:834:46: note: candidate function
inline _LIBCPP_INLINE_VISIBILITY float       asin(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:835:46: note: candidate function
inline _LIBCPP_INLINE_VISIBILITY long double asin(long double __lcpp_x) ...
                                             ^
libcxx-10.0.0.src/include/math.h:846:93: error: no member named
      'atanf' in the global namespace; did you mean 'atan'?
  ...float       atan(float __lcpp_x) _NOEXCEPT       {return ::atanf(__lcpp_...
                                                              ~~^
libcxx-10.0.0.src/include/math.h:846:46: note: 'atan' declared here
inline _LIBCPP_INLINE_VISIBILITY float       atan(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:847:93: error: no member named
      'atanl' in the global namespace; did you mean 'atan'?
  ...long double atan(long double __lcpp_x) _NOEXCEPT {return ::atanl(__lcpp_...
                                                              ~~^
libcxx-10.0.0.src/include/math.h:846:46: note: 'atan' declared here
inline _LIBCPP_INLINE_VISIBILITY float       atan(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:853:38: error: call to 'atan' is
      ambiguous
atan(_A1 __lcpp_x) _NOEXCEPT {return ::atan((double)__lcpp_x);}
                                     ^~~~~~
libcxx-10.0.0.src/include/math.h:846:46: note: candidate function
inline _LIBCPP_INLINE_VISIBILITY float       atan(float __lcpp_x) _NOEXC...
                                             ^
libcxx-10.0.0.src/include/math.h:847:46: note: candidate function
inline _LIBCPP_INLINE_VISIBILITY long double atan(long double __lcpp_x) ...
                                             ^
libcxx-10.0.0.src/include/math.h:858:116: error: no member named
      'atan2f' in the global namespace; did you mean 'atan2'?
  ...__lcpp_y, float __lcpp_x) _NOEXCEPT             {return ::atan2f(__lcpp_...
                                                             ~~^
libcxx-10.0.0.src/include/math.h:858:46: note: 'atan2' declared here
inline _LIBCPP_INLINE_VISIBILITY float       atan2(float __lcpp_y, floa...
                                             ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
```
</details>